admin/listPurchases/listView.html

```diff
@@ -407,6 +408,34 @@
     `;
 }
 
+function loadSomeOrders(){
+    requestOrdersWithQuery(createQueryString(), (error, orders) => {
+        if(error) {
+            return console.error("Failed to load entries", error);
+        }
+        entries = orders.orders;
+        amountOfEntries = orders.amountOfEntries;
+        options.find(o => o.cls == "option-new-orders").el.click();
+    })
+}
+
+async function requestOrdersWithQuery(queryString, callback){
```

Comment:
Naming: `withQuery` communicates only implementation details and is bad naming.

Your function names should communicate *what they do*, now how they are implemented.

You can make the `queryString` param optional by doing:

```js
async function requestOrders(queryString, callback) {
  if(!callback) {
    callback = queryString;
    queryString = "";
  }

  // ... rest of function
}
```

This way, you can call the function as `requestOrdersWithQuery((error, data) => ...)` or `requestOrdersWithQuery("hello=world", (error, data) => ...)` and both work